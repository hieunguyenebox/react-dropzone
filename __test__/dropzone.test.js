import React from 'react';
import Dropzone from '../lib';
import renderer from 'react-test-renderer';

test('ok fine', () => {
  const component = renderer.create(<Dropzone defaultStyle />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
