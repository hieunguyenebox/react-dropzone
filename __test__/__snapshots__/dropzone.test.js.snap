// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ok fine 1`] = `
<div
  className="dz-container dz-style"
  onDragEnter={[Function]}
  onDragLeave={[Function]}
  onDragOver={[Function]}
  onDrop={[Function]}
  style={Object {}}
>
  <div
    className="dz-img "
  />
  <span>
    Drop file here
  </span>
</div>
`;
